### DISCUSSION TITLE
INDI

### DISCUSSION THREAD

**Discussion by EricCabrol** [thomasfermi/Algorithms-for-Automated-Driving#9] [General]
As far as I know, INDI stands for Incremental Nonlinear Dynamic Inversion.
I found a lesson here : http://www.aerostudents.com/courses/advanced-flight-control/nonlinearDynamicInversion.pdf
INDI is mentioned in chapter 5

**Comment by thomasfermi** [comment]
Thank you very much! I will include that reference in the book! (probably tomorrow)

**Reply by thomasfermi** [reply]
Reference included. Should be reflected on the website in 10-20 minutes.

Thanks again :)
